content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,152 @@
+
+Additionally, we should mention [ARM TrustZone](https://www.arm.com/technologies/trustzone-for-cortex-a) which is optimized for embedded devices such as smartphones, tablets, and smart TVs, as well as [AWS Nitro Enclaves](https://aws.amazon.com/ec2/nitro/nitro-enclaves/) which are only available on [Amazon Web Services](https://aws.amazon.com/) and have a different attacker model compared to the CPU-based solutions by Intel and AMD.
+
+### Security Properties and Feature Set
+
+In the following, we will review the security properties and additional features these new technologies bring to the table. Not all solutions will provide all properties, and we will discuss each technology in further detail in their respective chapter.
+
+The **Confidentiality** property ensures that information cannot be viewed while it is in use in the TEE. This provides us with the highly desired feature to secure **data in use**. Depending on the specific TEE used, both code and data may be protected from outside viewers.
+
+Confidentiality is a great feature, but an attacker would still be able to manipulate or inject arbitrary code and data for the TEE to execute and therefore easily leak critical information. **Integrity** allows a TEE owner to check that neither code nor data has been tempered before running critical computations.
+
+*Availability* is the third basic property often [discussed in information security](https://en.wikipedia.org/wiki/Information_security). This property is outside the scope of most TEEs. Usually, they can be controlled (shutdown, restarted, ...) by some higher-level concept. This could be the CPU itself, a hypervisor, or a kernel. This is to preserve the availability of the overall system, and not the TEE itself. When running in the cloud the availability is usually guaranteed by the cloud provider in [terms of Service Level Agreements (SLAs)](https://cloud.google.com/compute/sla) and is not cryptographically enforceable.
+
+Confidentiality and Integrity by themselves would only be helpful in some cases. For example, consider a TEE running in a remote cloud. How would you know that the TEE is genuine and running your intended software? It could be an imposter stealing your data as soon as you send it over. This fundamental problem is addressed by **Attestability**. Remote attestation allows us to verify the identity, confidentiality, and integrity of TEEs based on cryptographic certificates issued from the hardware itself.
```

Comment:
Is the word remote appropriate in the last sentence?

Reply:
Good point, I've added a sentence to clarify.